tests/pipelines/deepfloyd_if/test_if.py

```diff
@@ -185,15 +138,12 @@ def _test_if(self, pipe_1, pipe_2, prompt_embeds, negative_prompt_embeds):
         )
         assert_mean_pixel_difference(image, expected_image)
 
-        # pipeline 2
-
+        # Super resolution test
         _start_torch_memory_measurement()
-
         generator = torch.Generator(device="cpu").manual_seed(0)
-
         image = floats_tensor((1, 3, 64, 64), rng=random.Random(0)).to(torch_device)
 
-        output = pipe_2(
+        output = pipe_super_res(
```

Comment:
I think we shouldn't do super-resolution here. 

Instead, it could make sense to pre-serialize the inputs and pass them accordingly to the pipeline for testing. This should reduce some timing, too. 

WDYT?